projects/opendr_ws_2/src/data_generation/data_generation/synthetic_facial_generation.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3.6
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cv2
+import torch
+import rclpy
+from ros2_bridge.bridge import ROS2Bridge
+import numpy as np
+from sensor_msgs.msg import Image as ROS_Image
+from SyntheticDataGeneration import MultiviewDataGeneration
+import os
+import argparse
+from opendr.engine.data import Image
+from algorithm.DDFA.utils.ddfa import str2bool
+
+
+class Synthetic_Data_Generation:
+
+    def __init__(self, input_image_topic="/usb_cam/image_raw", output_image_topic="/opendr/synthetic_facial_images",
```

Comment:
For consistency reasons across the ROS2 nodes the argument names should change. Also in ROS2 the default webcam package publishes `image_raw` topic and thus the default value of the input topic should change.

```suggestion
    def __init__(self, input_rgb_image_topic="image_raw", output_rgb_image_topic="/opendr/synthetic_facial_images",
```